source/_docs/configuration/customizing-devices.markdown

```diff
@@ -74,6 +69,12 @@ initial_state:
   type: string
 {% endconfiguration %}
 
+<div class='note'>
```

Comment:
Let's just remove it from here completely and just mention it in the breaking changes in the release notes.

It has been marked here as deprecated for ages already.

Reply:
Done